dotnet/src/IntegrationTests/Connectors/Memory/AzureAISearch/AzureAISearchVectorStoreFixture.cs

```diff
@@ -64,6 +66,11 @@ public AzureAISearchVectorStoreFixture()
                 new VectorStoreRecordDataProperty("Rating", typeof(double?))
             }
         };
+        OpenAIConfiguration? openAIConfig = s_configuration.GetSection("OpenAIEmbeddings").Get<OpenAIConfiguration>();
```

Comment:
Can we use AzureOpenAI instead?  We don't need api keys for that, so it's just easier to run.